### PR TITLE
Define logout, clarify ID Site redirects on login/logout

### LINKED ISSUE #43: Implement logout as POST
There is a strong argument that the logout URI should not respond to GET requests, because:
- Other sites can maliciously log you out via simple image tags.  E.g. http://superlogout.com/ :)
- The omnibar in the browser will pre-emptibely request the logout URI if it thinks you are typing it, and log you out.

More reading here: 

http://stackoverflow.com/questions/3521290/logout-get-or-post

As such, I believe we should switch to using POST instead.

### PR DESCRIPTION
**Pull request by robertjd**
Closes #43 #25

### CODE REVIEW

**Comment by rorheden** [comment]
Looks good! :+1:

File: logout.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Logout
+
+The integration must implement the `/logout` endpoint.  The endpoint is used to
+delete the [Authentication Cookies](cookie-authentication.md) that were set on
+login.
+
```
Comment (bretterer) [inline logout.md:7]: We should also make a note to remove access tokens and refresh tokens at stormpath for non Id site logout

**Comment by robertjd** [comment]
Thanks @bretterer RE deleting the tokens.

**Comment by lhazlewood** [comment]
@robertjd shouldn't we have something in here about CSRF?  We shouldn't allow requests to this endpoint without CSRF protection really...

**Pull request merged**